aces/visualization/figure_configuration.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from astropy import units as u
+import matplotlib.colors as mcolors
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import pylab as pl
+
+from aces.visualization.compare_to_other import format_ax
+
+pl.rcParams['figure.facecolor'] = 'w'
+pl.rcParams['figure.figsize'] = (10, 8)
+pl.rcParams['font.size'] = 16
+pl.rcParams['xtick.direction'] = 'in'
+pl.rcParams['ytick.direction'] = 'in'
+
+colors1 = pl.cm.gray_r(np.linspace(0., 1, 128))
+#colors2 = pl.cm.inferno(np.linspace(0, 1, 128))
+colors2 = pl.cm.hot(np.linspace(0, 1, 128))
+colors2b = pl.cm.afmhot(np.linspace(0, 1, 128))
+
+colors = np.vstack((colors1, colors2))
+mymap = mcolors.LinearSegmentedColormap.from_list('my_colormap', colors)
+
+colors_stack2 = np.vstack((colors1, colors2b))
+mymap2 = mcolors.LinearSegmentedColormap.from_list('my_afmcolormap', colors_stack2)
+
+distance = 8.1*u.kpc
```

Comment:
8.277 from Gravity

Reply:
I think the 2022 paper has a typo, the original paper they reference reports a distance of 8.178 kpc 
https://ui.adsabs.harvard.edu/abs/2019A%26A...625L..10G/abstract

Reply:
what's the 2022 paper link?

Reply:
https://ui.adsabs.harvard.edu/abs/2021A%26A...647A..59G/abstract gives 8.275

Reply:
Ooo so there is an updated distance measurement. I was wary because the paper linked was https://ui.adsabs.harvard.edu/abs/2022A%26A...657L..12G/abstract, which only mentions the distance in the introduction.

Reply:
The 2022 paper has an updated distance estimate of 8277 pc, in their Table B.1.